### PR TITLE
Payload logic

### PR DESCRIPTION
**Pull request by MrDuck557**
I added some sensing capability to payloads (items, liquids, totalItems, firstItem, totalLiquids)
also implemented forcing payload loaders/unloaders to push out their payload (control shoot), as well as a way to artificially cap the number of items that are loaded in (control config).

If your pull request is **not** translation or serverlist-related, read the list of requirements below and check each box:

- [X] I have read the [contribution guidelines](https://github.com/Anuken/Mindustry/blob/master/CONTRIBUTING.md).
- [X] I have ensured that my code compiles, if applicable.
- [X] I have ensured that any new features in this PR function correctly in-game, if applicable.

### CODE REVIEW

**Comment by MrDuck557** [comment]
dang it

**Comment by Anuken** [comment]
This improperly formats a bunch of unrelated code, please revert.

**Comment by MrDuck557** [comment]
one mo

File: core/src/mindustry/world/blocks/payloads/PayloadBlock.java [hunk 1]
```diff
@@ -147,7 +150,7 @@ public boolean blends(int direction){
         }
 
         public void updatePayload(){
-            if(payload != null){
+            if (payload != null) {
```
Comment (MEEPofFaith) [inline core/src/mindustry/world/blocks/payloads/PayloadBlock.java:153]: noooo change your ide settings

**Comment by Anuken** [comment]
(there are still formatting errors everywhere)

File: core/src/mindustry/world/blocks/payloads/PayloadBlock.java (hunk 1 above)
Comment (MrDuck557) [reply core/src/mindustry/world/blocks/payloads/PayloadBlock.java]: https://github.com/Anuken/Mindustry/pull/6271/commits/45acbcc740c79ef4b13f7b940404d9ecac8cbdb8

**Comment by Anuken** [comment]
image

You've still reformatted a bunch of unrelated stuff in an ugly way - just don't touch code that you're not changing.

**Comment by MEEPofFaith** [comment]
~~Don't let your IDE do things automatically~~

**Comment by MrDuck557** [comment]
> You've still reformatted a bunch of unrelated stuff in an ugly way

oh my god
can you open a review

**Comment by Anuken** [comment]
> can you open a review

I don't need to. Just open the [diff](https://github.com/Anuken/Mindustry/pull/6271/files) and observe that you've changed loads of formatting in unrelated code.

**Pull request closed without merging**